drivers/counter/Kconfig.stm32_rtc

```diff
@@ -75,6 +83,13 @@ config COUNTER_RTC_STM32_LSE_BYPASS
 	help
 	  Enable LSE bypass
 
+config COUNTER_RTC_STM32_HSE_DIV
+	int "HSE prescaler"
+	depends on COUNTER_RTC_STM32_CLOCK_HSE
+	default 0
```

Comment:
some stm32 devices  like stm32F4 have a prescaler from 2 to 31 on the HSE clock, some others like the stm32wb or stm32L4/L5 have a fixed divider by 32 on the HSE clock